cmd/checkmetrics/compare.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"strconv"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// metricsCheck is a placeholder struct for us to attach the methods to and make
+// it clear they belong this grouping. Maybe there is a better way?
+type metricsCheck struct {
+}
+
+// reportTitleSlice returns the report table title row as a slice of strings
+func (mc metricsCheck) reportTitleSlice() []string {
```

Comment:
Ah, that was a typo - nice spot. Now I have to go read up on what that actually does... and then fix it most likely